docs/static/logging.asciidoc

```diff
@@ -42,6 +42,11 @@ logger.elasticsearchoutput.name = logstash.outputs.elasticsearch
 logger.elasticsearchoutput.level = debug
 --------------------------------------------------
 
+In the previous example it's defined a name and a level for the logger `logstash.outputs.elasticsearch`.
+The logger is usually identified by a Java class name, for example in `org.logstash.dissect.Dissector` but could also a partial
+package path like in `org.logstash.dissect`. For Ruby classes, like `LogStash::Outputs::Elasticsearch`,
```

Comment:
```suggestion
The logger is usually identified by a Java class name, such as
`org.logstash.dissect.Dissector`, for example.  It can also be a partial package
path as in `org.logstash.dissect`.  For Ruby classes, like `LogStash::Outputs::Elasticsearch`,
```